.github/CONTRIBUTING.md

```diff
@@ -1,34 +1,90 @@
+
+
+## Getting Started
+
+Please ensure you follow [the Kitura coding standards](https://github.com/IBM-Swift/Kitura/blob/master/Documentation/CodeConventions.md). This keeps the source code tidy, and allows [Swift Lint]((https://github.com/realm/SwiftLint)) to work with the project.
+
+**If you are new:** We have created the label ["good starting issue"](https://github.com/IBM-Swift/Kitura/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) to help new members of the community get involved with Kitura quickly and effectively. These issues include documentation changes, test cases, minor bug fixes, logging, renaming and refactoring.
+
+To get started, you will need to open a Terminal and:
+
+1. Fork this repo and clone it onto your machine.
+
+   `$ git clone https://github.com/YOUR_GITHUB_ID/Kitura`
+
+
+2. Make changes to code, usually by tackling an issue. A list of issues can be found at [here](https://github.com/IBM-Swift/Kitura/issues). If you are new to Swift or software development, look for issues labelled ["good starting issue"](https://github.com/IBM-Swift/Kitura/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).
+
+   If there aren't any tagged issues, post in [Slack](http://swift-at-ibm-slack.mybluemix.net/) and the team would be happy to help you get started.
+
+   **Tip:** If you are on macOS, Xcode can be a great way to work on Kitura. You can download the latest Xcode [here](https://itunes.apple.com/us/app/xcode/id497799835?ls=1&mt=12). You'll need to open a Terminal window, and navigate to the root of the Kitura project you cloned. Then, type the following command into Terminal:
+
+   `$ swift package generate-xcodeproj`
+   ​
+   This will generate an Xcode Project, open the .xcodeproj file with:
+
+   `$ open *.xcodeproj`
+
+   *Note: Xcode is macOS only, Linux users will need to use a different editor.*
+
+3. All source code submitted requires an Apache License header at the top of the file. A Github Gist of this text can be found [here](https://gist.github.com/SwiftDevOps/141437c6861f88c959d0731bc3b16bee), just copy and paste it at the top of any new files you're submitting.
+
+4. Ensure all tests pass with your changes by running `swift test` in Terminal. If there is any new functionality introduced by your changes, new test case(s) may be required. If you need any help writing tests, contact us on [Slack](http://swift-at-ibm-slack.mybluemix.net/).
+
+5. If they all pass, open a Pull Request following the guidelines below.
+
+
+## Pull Requests
+
+**Note about tests:** When opening a Pull Request, please run the tests for your OS with `swift test`.  Kitura is a cross platform web framework that uses Continuous Integration. You may see mentions of Travis CI in your PR and it's comments. Travis is an external service we use that is automatically called on every PR, and runs the test cases on macOS and Linux to ensure every code change is cross plaform compatible.
```

Comment:
One of these links should actually link to Travis.

Reply:
✅